teorija.tex

```diff
@@ -3302,8 +3301,8 @@ \subsection{Skalarni produkt}
 
 Dokaz lastnosti~\ref{eq:vec:skal:aajea2}:
 \begin{align*}
-  &\vec{a}\vec{a} = a\krat a \krat\kos0\deg = a^2 \\
-  &a = \sqrt{\vec{a}\vec{a}} \comment{Formula za dolžino vektorja.}
+  &\vec{a}\vec{a} = a\krat a \krat\kos0\deg = |\vec{a}|^2 \\
+  &|\vec{a}| = \sqrt{\vec{a}\vec{a}} \comment{Formula za dolžino vektorja.}
```

Comment:
Mi smo vedno pisali s puščicami. To lahko tudi vidiš iz teh vaj: http://ucilnica.gimvic.org/pluginfile.php/2084/mod_resource/content/1/Vektorjimatura.pdf. Pa tudi drugod po internetu: https://eucbeniki.sio.si/vega2/260/index5.html, http://www2.arnes.si/~mpavle1/mp/vektor.html.